src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/MultiNodeTreePickerConverter.cs

```diff
@@ -100,7 +100,9 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
                 return multiNodeTreePicker;
             }
 
-            return base.ConvertFrom(context, culture, value);
+            // [ML] - return default list as its always nice to avoid null reference exceptions on lists =)
```

Comment:
Cool, I think the base converters throw exceptions as values passed in are always strings from archetype, so some are unable to cast to the new type a flake out. Sure there will be some work around we can use.